keys-in.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @module 101/keys-in
+ */
+
+/**
+ * Get all the keys compositing an object, including the `Object.prototype`
+ * @function module:101/keys-in
+ *
+ * @param {object} object - the object from which to extract the keys
+ * @return {array} array of keys
+ */
+module.exports = keysIn;
+
+function keysIn(object) {
+  if (!object) { return []; }
+
+  var keys = [];
+
+  if (object.prototype) { object = Object.create(object.prototype); }
```

Comment:
I understand it creates an instance of the object passed if it has a prototype (is a class). 
Although I am not sure if I would infer that behavior as a user.